education.html

```diff
@@ -22,49 +22,44 @@ <h2 class="hero__subtitle">Here is where I had opportunities to learn!</h2>
 </main>
 
 <section class="education">
-  <div class="education__section">
-    <img class="content__img" src="https://www.codecademy.com/assets/logo-codecademy-social-abfd8450722d675bddedde689f8af624.png">
-    <h1 class="content__title">Codecademy</h1><br>
-    <ul class="content__list">
-      <li class="content__listitem">Introduction to HTML</li>
-      <li class="content__listitem">Learn CSS</li>
-      <li class="content__listitem">Make a Website</li>
+  <img class="education__img1" src="https://www.codecademy.com/assets/logo-codecademy-social-abfd8450722d675bddedde689f8af624.png">
```

Comment:
`__img1` should be more descriptive. So `__img-education` would be more descriptive